src/uct/api/tl.h

```diff
@@ -145,6 +146,8 @@ typedef struct uct_iface_ops {
 
     void         (*ep_pending_purge)(uct_ep_h ep, uct_pending_callback_t cb);
```

Comment:
I'd not, because we have to make unnecessary pending list operations otherwise, which (potentially) affects recovery time.